jujugui/static/gui/src/app/components/added-services-list/test-added-services-list.js

```diff
@@ -308,26 +287,23 @@ describe('AddedServicesList', () => {
           services={services}/>, true);
 
     var instance = renderer.getMountedInstance();
-    // Call the fade Service method which would be passed down to the children.
-    instance.fadeService('trusty/wordpress');
-    assert.equal(serviceSet.callCount, 1);
-    assert.deepEqual(serviceSet.args[0], ['fade', true]);
+    var unfocusService = sinon.stub(instance, 'unfocusService');
+    instance.fadeService(id);
+    // Set the fade switch appropriately.
+    assert.equal(allServices[0].get('fade'), true);
+    // Unfocused the service.
+    assert.isTrue(unfocusService.calledWith(id));
+    unfocusService.restore();
```

Comment:
I'm not sure if this is actually a problem with these tests....but if there is a failure between when this is stubbed out and when it's restored, it will never be restored. Could this be tested without stubbing that fn to avoid any potential issues?